docs/rebuilding.md

```diff
@@ -0,0 +1,70 @@
+# Rebuilding Packages & Creating the ConnectBox repository
+
+Due to architectural requirements of some models of the Raspberry Pi, it is necessary to ocassionally rebuild some of the packages used by the ConnectBox (see [Raspberry_Pi_issues](https://wiki.debian.org/RaspberryPi#Raspberry_Pi_issues) for details) and host them in our own repository.
+
+Complete documentation on how to build a package for Debian can be found at their [BuildingTutorial](https://wiki.debian.org/BuildingTutorial), so the steps will only be summarized here.
+
+## Install the toolchain
+
+The easiest way to build compatible packages is directly on the Raspberry Pi 3 (The alternative is going through the effort of setting up a cross compiler toolchain on faster hardware).
+
+    sudo apt-get install build-essential fakeroot devscripts
+
+## Set up a source repository
+
+The robust Debian ecosystem allows us to reproducibly rebuild almost any package without significant effort. One aspect of this is that every major package repository has a cooresponding source repository. For these instructions, add the jessie-backports source repo to apt:
+
+    echo 'deb-src http://ftp.debian.org/debian jessie-backports main contrib' > /etc/apt/sources.list.d/jessie-backports-src.list
+
+## Rebuild the package
+
+Make a directory to work in (the name is not important)
+
+    mkdir debian
+    cd debian
+
+Get the source code from the src repo
+
+    apt-get apt-get source libssl-dev
```

Comment:
Ah ha (referencing issue below), you don't need sudo apt-get source, just apt-get source. My mistake there with the typo though.